pkg/controller/webhook/webhook_controller.go

```diff
@@ -0,0 +1,170 @@
+package webhook
```

Comment:
This is not a controller and I don't think it belongs under `pkg/controller`. Just `pkg/webhook` should be fine.

Reply:
Fixed.